erizo/src/erizo/WebRtcConnection.h

```diff
@@ -208,6 +216,53 @@ class WebRtcConnection: public TransportListener, public LogContext,
   bool first_remote_sdp_processed_;
 
   std::unique_ptr<BandwidthDistributionAlgorithm> distributor_;
+  Pipeline::Ptr pipeline_;
+  bool pipeline_initialized_;
+  std::shared_ptr<HandlerManager> handler_manager_;
+};
+
+class ConnectionPacketReader : public InboundHandler {
+ public:
+  explicit ConnectionPacketReader(WebRtcConnection *connection) : connection_{connection} {}
+
+  void enable() override {}
+  void disable() override {}
+
+  std::string getName() override {
+    return "connection-reader";
```

Comment:
This makes me think we could, at some point, change the mediaStream equivalent's name to "mediastream-reader"